modules/solid_mechanics/include/materials/CombinedCreepPlasticity.h

```diff
@@ -23,14 +23,12 @@ class CombinedCreepPlasticity : public ConstitutiveModel
 
   /// Compute the stress (sigma += deltaSigma)
   virtual void computeStress(const Elem & current_elem,
-                             unsigned qp,
                              const SymmElasticityTensor & elasticityTensor,
                              const SymmTensor & stress_old,
                              SymmTensor & strain_increment,
                              SymmTensor & stress_new);
 
   virtual bool modifyStrainIncrement(const Elem & current_elem,
-                                     unsigned qp,
```

Comment:
This still isn't completely foolproof because the model that call it has to set the qp before it calls it, but that only happens in a couple of places in the code, and matches the tensor_mechanics pattern.  In the models themselves, you just always use _qp now, which I think is easier.